src/hub.html

```diff
@@ -329,6 +328,7 @@
 
                 <template data-selector=".Neck">
                     <a-entity>
+                        <a-entity personal-space-bubble="radius: 1" position="0 -0.3 0"></a-entity>
```

Comment:
Since I'm looking up when I toggle the personal space bubble, I only see (roughly) the second half of the bubble's animation because it starts below my camera and grows outward. I wonder if there's a way (with a cylinder, or something jim makes, or a different center point) to make this look better assuming the user will be looking up when they are discovering what it is.